packages/stream-model/src/model.ts

```diff
@@ -118,6 +118,10 @@ export class Model extends Stream {
     return super.content
   }
 
+  get metadata(): ModelMetadata {
+    return { controller: this.state$.value.metadata.controllers[0] }
```

Comment:
Should we also return the model property here?

Reply:
I'm worried that if we expose it, users will try to use the streamid to load a stream and then be confused when it doesn't work.